build.gradle.kts

```diff
@@ -144,8 +144,10 @@ subprojects {
         val existingFile = file("$buildDir/resources/main/metadata.properties")
         existingFile.delete()
       }
-      // This is to replace the %APP_VERSION_TOKEN% in the  metadata.properties file.
-      filter { line -> line.replace("%APP_VERSION_TOKEN%", rootProject.version.toString()) }
+      filesMatching("**/metadata.properties") {
```

Comment:
without matching, the `jks` files are filtered and modified which causes the tests to fail.